types/src/cryptography/sig_hash.rs

```diff
@@ -0,0 +1,12 @@
+use crate::cryptography::Hash;
+
+pub const SIG_PREFIX_GAS_PRICE_ATTESTATION: &[u8] = b"ATT_GAS_PRICE";
+pub const SIG_VERSION_GAS_PRICE_ATTESTATION: u8 = 1;
```

Comment:
Do we need to separate prefix and version? Isn't it equal to have the version in the prefix like:
- `"ATT_GAS_PRICE"` for version 0
- `"ATT_GAS_PRICE_1"` for version 1 

and so on?